### PR TITLE
QA

### PR DESCRIPTION
**Pull request by kcbreuil**
update contact page and add character count on service/fleet cards

### CODE REVIEW

**Comment by kcbreuil** [comment]
@asasmith other QA items are in the other two open PRs

File: studio/schemas/documents/imageWithText.js [hunk 1]
```diff
@@ -19,5 +19,10 @@ export default {
             title: 'Image',
             type: 'mainImage',
         },
+        {
+            name: 'id',
+            title: 'Id',
+            type: 'string',
+        },
```
Comment (asasmith) [inline studio/schemas/documents/imageWithText.js:26]: What is this field for?

Comment (kcbreuil) [reply studio/schemas/documents/imageWithText.js:26]: anchor links, the logic is on the FE and i forgot to add it in sanity

Comment (asasmith) [reply studio/schemas/documents/imageWithText.js:26]: So how would this work? A user would have to input the correct string into the sanity field? Or are you planning to add these programmatically?

Comment (kcbreuil) [reply studio/schemas/documents/imageWithText.js:26]: @asasmith I guess not, I can hardcode the anchor link instead

**Review by asasmith — APPROVED** [review]

**Pull request merged**